test/test_db/test_db.py

```diff
@@ -247,6 +247,51 @@ def test_function_create_new_user_group(self):
         self.assertEqual(row_data_source_policy[0], 'SELECT')
         self.assertEqual(row_data_source_policy_admin[0], 'ALL')
 
+    def test_function_get_all_user_groups(self):
+        # Insert test get all user group roles
+
+        # Prepare database, delete all user group roles in the pg_authid table
+        delete_user_groups_roles = f'''DELETE FROM pg_authid WHERE rolname LIKE 'user_group_%';'''
+
+        # Create new user groups
+        create_test_user_group_standard = f'''CREATE ROLE user_group_mobydq;'''
+        self.connection.execute(create_test_user_group_standard)
+        create_test_user_group_admin = f'''CREATE ROLE user_group_mobydq_admin;'''
+        self.connection.execute(create_test_user_group_admin)
+
+        # Call tested function to get all user groups
+        call_test_get_all_user_groups = f'''SELECT base.get_all_user_groups();'''
+        cursor = self.connection.execute(call_test_get_all_user_groups)
+        user_group_list = list(cursor)
+
+        # Assert all created user groups are in the returned list
+        self.assertTrue('user_group_mobydq' in [item[0] for item in user_group_list])
```

Comment:
Don't do the same list comprehension twice. Just put it into a variable.

Reply:
I changed to variable